Modix.Bot/Modules/IlModule.cs

```diff
@@ -102,6 +111,8 @@ private async Task<EmbedBuilder> BuildEmbed(SocketGuildUser guildUser, string co
 
             await embed.UploadToServiceIfBiggerThan(result, "asm", 990, _pasteService);
 
+            embed.WithFooter(a => a.WithText("React with ❌ to remove this embed."));
```

Comment:
Why are you doing that instead of this? 
`embed.WithFooter("React with ❌ to remove this embed.");`

Reply:
My imperfection is showing... 😬